asm/control-plane-revision/crd.yaml

```diff
@@ -15,88 +15,150 @@ spec:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ControlPlaneRevisionSpec defines the desired state of ControlPlaneRevision
+            properties:
+              channel:
+                description: ReleaseChannel determines the aggressiveness of upgrades.
+                enum:
+                - regular
+                - rapid
+                - stable
+                type: string
+              type:
+                description: ControlPlaneRevisionType determines how the revision
+                  should be managed.
+                enum:
+                - managed_service
+                type: string
+            type: object
+          status:
+            description: ControlPlaneRevisionStatus defines the observed state of
+              ControlPlaneRevision.
+            properties:
+              conditions:
+                items:
+                  description: ControlPlaneRevisionCondition is a repeated struct
+                    definining the current conditions of a ControlPlaneRevision.
+                  properties:
+                    lastTransitionTime:
+                      description: Last time the condition transitioned from one status
+                        to another
+                      format: date-time
+                      type: string
+                    message:
+                      description: Human-readable message indicating details about
+                        last transition
+                      type: string
+                    reason:
+                      description: Unique, one-word, CamelCase reason for the condition's
+                        last transition
+                      type: string
+                    status:
+                      description: Status is the status of the condition. Can be True,
+                        False, or Unknown.
+                      type: string
+                    type:
+                      description: Type is the type of the condition.
+                      type: string
+                  type: object
+                type: array
+            type: object
+        type: object
+    served: true
+    storage: true
```

Comment:
Just to verify, this is consistent with what AFC installs?

Reply:
Yes. This is copied from the file generated using the blaze command.